ddtrace/ext/test.py

```diff
@@ -32,8 +32,8 @@
 # Test Parameters
 PARAMETERS = "test.parameters"
 
-# Pytest Result (XFail, XPass)
-RESULT = TEST_RESULT = "pytest.result"
+# Test Result (XFail, XPass)
+RESULT = TEST_RESULT = "test.result"
```

Comment:
I think this is pretty harmless but please double-check with backend/core/app folks to make sure we're not using `pytest.result` elsewhere.

https://github.com/search?q=org%3ADataDog%20pytest.result&type=code suggests we're not, but "famous last words".